website/docs/configuration.md

```diff
@@ -404,46 +404,76 @@ corner.
 
 ### Input Settings
 
-#### Use Logo Key
+#### macOS Alt is Meta
 
 VimScript:
 
 ```vim
-let g:neovide_input_use_logo = v:false  " v:true on macOS
+let g:neovide_input_macos_alt_is_meta = v:false
 ```
 
 Lua:
 
 ```lua
-vim.g.neovide_input_use_logo = false    -- true on macOS
+vim.g.neovide_input_macos_alt_is_meta = false
 ```
 
-Setting `g:neovide_input_use_logo` to a boolean value will change how logo key (also known as
-[super key](<https://en.wikipedia.org/wiki/Super_key_(keyboard_button)>),
-[command key](https://en.wikipedia.org/wiki/Command_key) or
-[windows key](https://en.wikipedia.org/wiki/Windows_key)) is handled, allowing all key combinations
-containing logo to be forwarded to neovim. On MacOS, this defaults to `true` (so that e.g. `cmd+v`
-works for pasting with respective setup of `init.vim`), and to `false` for other platforms (that
-typically use e.g. `ctrl+v` for pasting).
+**Available since 0.10.**
 
-#### macOS Alt is Meta
+Interprets <kbd>Alt</kbd> + <kbd>whatever</kbd> actually as `<M-whatever>`, instead of sending the
+actual special character to Neovim.
+
+#### IME
 
 VimScript:
 
 ```vim
-let g:neovide_input_macos_alt_is_meta = v:false
+let g:neovide_ime = v:false
 ```
 
 Lua:
 
 ```lua
-vim.g.neovide_input_macos_alt_is_meta = false
+vim.g.neovide_input_ime = false
```

Comment:
The values shown should match the defaults, and in `src/window/settings.rs` the default seems to be `true`, not `false`.